app/models/observation/scopes.rb

```diff
@@ -330,7 +330,9 @@ module Observation::Scopes # rubocop:disable Metrics/ModuleLength
         left_outer_joins(:location).gps_in_box(box).
           or(Observation.associated_location_center_in_box(box))
       else
-        gps_in_box(box).or(Observation.cached_location_center_in_box(box))
+        gps_in_box(box).or(
+          Observation.cached_location_center_in_box(box).distinct
```

Comment:
Do you need `.distinct` here? It's now in `cached_location_center_in_box`
```suggestion
          Observation.cached_location_center_in_box(box)
```

Reply:
Shouldn't then.  I'll leave that to you, oh interested reader!